operator/konflux.bundle.Dockerfile

```diff
@@ -14,13 +14,35 @@ RUN make bundle-post-process
 
 FROM scratch
 
+ARG MAIN_IMAGE_TAG
+
+# Enterprise Contract labels.
+LABEL com.redhat.component="rhacs-operator-bundle-container"
+LABEL com.redhat.license_terms="https://www.redhat.com/agreements"
+LABEL description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL distribution-scope="public"
+LABEL io.k8s.description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL io.k8s.display-name="operator-bundle"
+LABEL io.openshift.tags="rhacs,operator-bundle,stackrox"
+LABEL maintainer="Red Hat, Inc."
+LABEL name="rhacs-operator-bundle"
+LABEL source-location="https://github.com/stackrox/stackrox"
+LABEL summary="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c"
+LABEL vendor="Red Hat, Inc."
+# We must set version label to prevent inheriting value set in the base stage.
```

Comment:
[nit] The base stage is `scratch` so it has no version label. Let's remove this comment.
```suggestion
```

I saw this before but realized just now.